app/controllers/concerns/filter_by_current_user_roles.rb

```diff
@@ -5,6 +5,8 @@ module FilterByCurrentUserRoles
     before_action :add_roles_to_filter_params!, only: :index
   end
 
+  # Filter controlled_resources by what role you have onto them,
```

Comment:
I'd prefer to rename the method to make this more implicit than add the comments...but i'm like a 2/10 on this.